packages/server-boilerplate/src/utils/initialiseApiClient.ts

```diff
@@ -21,7 +21,7 @@ const upsertUserAccount = async ({
   password: string;
   salt: string;
 }): Promise<string> => {
-  const passwordHash = encryptPassword(password, salt);
+  const passwordHash = await encryptPassword(password, salt);
```

Comment:
Whenever the app re-starts the apiClient hashes get re-created so they don't need to be migrated in a migration.